src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -155,4 +155,44 @@ private static RequestAndRelatedRecords setRequestQueuePosition(
   private static boolean isTheSameRequester(RequestAndRelatedRecords it, Request that) {
     return Objects.equals(it.getUserId(), that.getUserId());
   }
+
+  private Result<RequestAndRelatedRecords> refuseWhenUserHasAlreadyBeenLoanedItem(Result<RequestAndRelatedRecords> result,
+      LoanRepository loanRepository) {
+
+    RequestAndRelatedRecords requestAndRelatedRecords = result.value();
+
+    if(requestAndRelatedRecords != null) {
+      Request request = requestAndRelatedRecords.getRequest();
+    
+      Loan loan = null;
+      try {
+        loan = loanRepository
+                .findOpenLoanForRequest(request)
+                .get()
+                .value();
+      } catch (InterruptedException e) {
+        log.error(e.getMessage(), e);
+        Thread.currentThread().interrupt();
+      } catch (ExecutionException e) {
+        log.error(e.getMessage(), e);
+        Map<String, String> parameters = new HashMap<>();
+        String message = "The loan associated with this request could not be validated.";
+        return failedValidation(new ValidationError(message, parameters));
+      }
+  
+      String userId = request.getProxyUserId() == null ? request.getUserId() : request.getProxyUserId();
```

Comment:
My understanding is that the check in this change is that the item is already on loan to the requesting user.

And, I believe the user who is making the request is always referenced in the userId.

Why is the proxy user involved?